### DISCUSSION TITLE
[Bookmarks] Rename "List Settings" to "Rename"

### DISCUSSION THREAD

**Discussion by Shohreh** [organicmaps/organicmaps#1483] [Ideas]
Hello,

In the Bookmarks section, I suggest renaming the "List Settings" to… "Rename".

Cheers,

![Screenshot_2021-10-16-19-59-18-722_app organicmaps](https://user-images.githubusercontent.com/1541861/139560967-48e1c7c8-f3ab-4d04-9867-7a43ca25ec22.jpg)

**Comment by biodranik** [comment]
Thanks! See #1485
